projects/extension/sql/incremental/018-drop-foreign-key-constraint.sql

```diff
@@ -0,0 +1,54 @@
+do language plpgsql $block$
+DECLARE
+    _vectorizer RECORD;
+    _constraint_name text;
+    _sql text;
+BEGIN
+    -- Loop through all vectorizers
+    FOR _vectorizer IN 
+        SELECT 
+            v.id,
+            v.target_schema,
+            v.target_table,
+            v.source_schema,
+            v.source_table
+        FROM ai.vectorizer v
+    LOOP
+        -- Find the foreign key constraint for this vectorizer's store table
+        SELECT conname INTO _constraint_name
```

Comment:
in theory there could be multiple foreign key constraints here, but this would only happen if the users did so manually, and that would be very weird. so, i'm okay with the assumption